pkg/util/tests/kubectl/kubectl.go

```diff
@@ -317,6 +317,7 @@ func nsflag(ns string) string {
 // DeleteCRDs deletes the CRDs by given list of crds names
 func DeleteCRDs(crds []string) error {
 	for _, crd := range crds {
+		fmt.Printf("Deleting CRD: %s\n", crd)
```

Comment:
I don't think we should be using `fmt.Print` to log in tests. We should be using the GinkgoWriter or `By`. Ginkgo doesn't log anything if the test succeeds. By logging via `fmt.Print`, the test will always produce output.

Reply:
This was for debugging the error. I’ll delete this